src/time.scss

```diff
@@ -7,31 +7,221 @@ $block: #{$fd-namespace}-time;
 .#{$block} {
   @include fd-reset();
 
-  width: 184px; // Quick fix to solve overflow issue
+  $fd-time-item-height: 2.875rem;
```

Comment:
should be 2 rem for cozy mode, it only gets bigger for smaller applications like tablet and phone

Reply:
Hello @stefan-sto compact mode has 2rem, default one has 2.875rem

Reply:
@JKMarkowski I think this is the wrong Stefano ;)